src/psyclone/errors.py

```diff
@@ -39,6 +39,37 @@
 ''' This module provides various error classes used in PSyclone'''
 
 
+class LazyString:
```

Comment:
Add class to the `__all__` list at the bottom.

Reply:
Oops.